packages/headless/src/app/commerce-ssr-engine/factories/build-factory.ts

```diff
@@ -125,6 +126,21 @@ function buildSSRCommerceEngine(
   };
 }
 
+function fetchActiveRecommendationControllers(
```

Comment:
+0, but we could rename the function to `fetchEnabledRecommendationControllers`.